documentation/0018-how-to-use-npm-package/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: How to use or execute a package installed using npm
+description: 'How to include and use in your code a package installed in your node_modules folder'
+author: flaviocopes
+---
+
+When you install using `npm` a package into your `node_modules` folder, or also globally, how do you use it in your Node code?
+
+Say you install `lodash`, the popular JavaScript utility library, using
+
+```bash
+npm install lodash
+```
+
+This is going to install the package in the local `node_modules` folder.
+
+To use it in your code, you just need to import it into your program using `require`:
+
+```js
+const _ = require('lodash)
```

Comment:
Shouldn't this be `const _ = require('lodash')`
The closing quote

Reply:
Good catch!